web/html/src/components/input/InputBase.tsx

```diff
@@ -173,7 +177,10 @@ export class InputBase<ValueType = string> extends React.Component<InputBaseProp
     }
 
     if (!this.props.disabled && (value || this.props.required)) {
-      if (this.props.required && !value) {
+      const noValue = Array.isArray(this.props.name) ?
+        value == null || Object.values(value).filter(v => v).length === 0 :
```

Comment:
Using `==` is a linter warning, and should ideally be an error instead (but I need to clean up all instances first). I know it's convenient to use to check for both `undefined` and `null` in one go, but the problem is it isn't clear whether that's the intent or it's just a typo.  
I would recommend using `isNil` from lodash instead in this case.

Reply:
ACK